src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Elasticsearch/BulkUpdateProductsInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Elasticsearch;
+
+use Akeneo\Pim\Automation\DataQualityInsights\Domain\ValueObject\ProductIdCollection;
+
+/**
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface BulkUpdateProductsInterface
```

Comment:
As we are updating only the quality scores properties in the product and product model index what do you think about the following name?

```suggestion
interface BulkUpdateProductQualityScoresInterface
```